### ISSUE TITLE
Rubocop failing to run after upgrade to 0.34.0 from 0.33.0

### ISSUE THREAD

**Issue by EddyLuten** [bbatsov/rubocop#2230]
I'm getting the following error for all my projects when I bump the version number to 0.34.0 from 0.33.0:

```
Running RuboCop...
Inspecting 204 files


0 files inspected, no offenses detected
"\xFB" from ASCII-8BIT to UTF-8
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/result_cache.rb:33:in `write'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/result_cache.rb:33:in `block in save'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/result_cache.rb:29:in `open'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/result_cache.rb:29:in `save'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:89:in `process_file'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:57:in `block in inspect_files'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:55:in `each'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:55:in `inspect_files'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:33:in `run'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/cli.rb:26:in `run'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/rake_task.rb:50:in `run_cli'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/rake_task.rb:38:in `run_main_task'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/rake_task.rb:30:in `block (2 levels) in initialize'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/file_utils_ext.rb:58:in `verbose'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rubocop-0.34.0/lib/rubocop/rake_task.rb:26:in `block in initialize'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task.rb:240:in `call'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task.rb:240:in `block in execute'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task.rb:235:in `each'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task.rb:235:in `execute'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task.rb:179:in `block in invoke_with_call_chain'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/monitor.rb:211:in `mon_synchronize'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task.rb:172:in `invoke_with_call_chain'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/task.rb:165:in `invoke'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:150:in `invoke_task'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:106:in `block (2 levels) in top_level'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:106:in `each'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:106:in `block in top_level'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:115:in `run_with_threads'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:100:in `top_level'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:78:in `block in run'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:176:in `standard_exception_handling'
/Users/eluten/.rbenv/versions/2.2.2/lib/ruby/2.2.0/rake/application.rb:75:in `run'
/Users/eluten/.rbenv/versions/2.2.2/bin/rake:33:in `<main>'
RuboCop failed!
```

I can provide the .rubocop.yml files on request if so required.

**Comment by wli** [comment]
@EddyLuten - Looks like a problem in caching code. You can temporarily run rubocop with the `--cache false` command line option, or use this snippet in your `.rubocop.yml` file.

```
AllCops:
  UseCache: false
```

**Comment by wli** [comment]
I think @jonas054 worked on this, so he might have some insight as to why this is failing.

**Comment by MoveIncEddyLuten** [comment]
Thanks, @wli. We've locked in Rubocop to 0.33.0 this morning as a temporary workaround.

**Comment by EddyLuten** [comment]
Thanks, @wli. We've locked in Rubocop to 0.33.0 this morning as a temporary workaround.

**Comment by jonas054** [comment]
Duplicate of #2213 (and #2226 and #2229), fixed in #2221. So, fixed on `master` but not released as a gem version yet.
